sys/kern/src/syscalls.rs

```diff
@@ -60,19 +66,25 @@ pub unsafe extern "C" fn syscall_entry(nr: u32, task: *mut Task) {
     arch::with_task_table(|tasks| {
         // Work out the task index based on the pointer into the task table
         // slice. We could store the index *and* the pointer in globals,
-        // avoiding this divde, but divides are pretty cheap....
+        // avoiding this divide, but divides are pretty cheap....
```

Comment:
Not required for this PR, but what would be the advantage of storing the index in globals, vs including the index in the `struct Task` itself (and populating it when building the task table in kernel startup)?